docs/sample_config.yaml

```diff
@@ -1392,7 +1392,11 @@ sso:
     # phishing attacks from evil.site. To avoid this, include a slash after the
     # hostname: "https://my.client/".
     #
-    # By default, this list is empty.
+    # If public_baseurl is set, then the login fallback page (used by clients
+    # that don't have full support for SSO) is always included in this list.
+    #
+    # By default, this list is empty, except if public_baseurl is set (in which
+    # case the login fallback page is the only element in the list).
```

Comment:
this seems like a complicated way to explain this.

How about: 

> If public_baseurl is set, then the login fallback page (used by clients
> that don't natively support the required login flows) is whitelisted in addition
> to any URLs listed here.
>
> By default, this list is empty.

Reply:
or possibly reverse those two paragraphs, which I think makes it clearer which "list" we're talking about.

Reply:
Done (with a slight re-phrasing) in https://github.com/matrix-org/synapse/pull/7153/commits/63aea691a761a9b6a2058b54792fc5859e12cfba